clang/bindings/python/tests/cindex/test_location.py

```diff
@@ -16,6 +16,8 @@
 
 from .util import get_cursor, get_tu
 
+INPUTS_DIR = os.path.join(os.path.dirname(__file__), "INPUTS")
```

Comment:
I think our minimal requirements for Python version allow us to take advantage of `pathlib`.